introduction/HANDSON.md

```diff
@@ -226,3 +267,52 @@ spec:
   selector:
     app: nginx
 ```
+
+# Namespaces
+
+```yaml
```

Comment:
You are using JSON for the file format but specifying yaml as the markdown format, is that expected?

Reply:
You're right. I've just changed them to yaml